tests/test_security_path_traversal.py

```
@@ -204,3 +204,68 @@ def test_run_tailer_path_validation(tmp_path):
+
+@pytest.fixture
+def mock_managers(monkeypatch, tmp_path):
+    # Mock manager methods depending on where tests import them
+    monkeypatch.setattr("swarm.api.routes.wisdom._get_runs_root", lambda: tmp_path / "runs")
+    monkeypatch.setattr("swarm.api.routes.wisdom._get_repo_root", lambda: tmp_path / "repo")
+    monkeypatch.setattr("swarm.api.routes.evolution._get_runs_root", lambda: tmp_path / "runs")
+    monkeypatch.setattr("swarm.api.routes.evolution._get_repo_root", lambda: tmp_path / "repo")
+    return tmp_path
+
+@pytest.mark.asyncio
+async def test_get_wisdom_artifacts_path_traversal(mock_managers):
```

Ruff (0.15.4)

[warning] Unused function argument: `mock_managers`

(ARG001)